spark-indexer/src/main/java/org/uniprot/store/spark/indexer/chebi/GraphMergeVerticeMapper.java

```diff
@@ -0,0 +1,49 @@
+package org.uniprot.store.spark.indexer.chebi;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.uniprot.core.cv.chebi.ChebiEntry;
+import org.uniprot.core.cv.chebi.impl.ChebiEntryBuilder;
+import org.uniprot.core.util.Utils;
+
+import scala.Function2;
+
+class GraphMergeVerticeMapper
```

Comment:
-> GraphMergeVertexMapper (if you want to talk about only 1 vertex)

OR

-> GraphMergeVerticesMapper (for multiple vertices)